code/framework/src/scripting/resource/resource.cpp

```
@@ -0,0 +1,525 @@
+            _loadTimestamp     = other._loadTimestamp;
+            _environment       = std::move(other._environment);
+            _eventHandlers     = std::move(other._eventHandlers);
+            _exports           = std::move(other._exports);
+            _serverScriptPaths = std::move(other._serverScriptPaths);
+            _clientScriptPaths = std::move(other._clientScriptPaths);
+            _restartAttempts   = std::move(other._restartAttempts);
+
+            other._manifestValid = false;
+            other._state         = ResourceState::Unloaded;
+        }
+        return *this;
```

Cppcheck (2.18.0)

[error] failed to evaluate #if condition

(preprocessorErrorDirective)